script/build

```diff
@@ -41,7 +41,7 @@ up_to_date() {
 
 build_hub() {
   setup_gopath
-  [ -n "$1" ] && (up_to_date "$1" || go build -ldflags "-X github.com/github/hub/commands.Version `./script/version`" -o "$1")
+  [ -n "$1" ] && (up_to_date "$1" || go build -ldflags "-X github.com/github/hub/commands.Version=`./script/version`" -o "$1")
```

Comment:
Note that potential conflict with this PR: https://github.com/github/hub/pull/861